src/Common/src/Interop/Ole32/Interop.CreateILockBytesOnHGlobal.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal static partial class Ole32
+    {
+        [DllImport(Libraries.Ole32, PreserveSig = false)]
+        public static extern ILockBytes CreateILockBytesOnHGlobal(IntPtr hGlobal, bool fDeleteOnRelease);
```

Comment:
@JeremyKuhne do we need decorate the `bool` here?
```suggestion
        public static extern ILockBytes CreateILockBytesOnHGlobal(IntPtr hGlobal, [MarshalAs(UnmanagedType.Bool)] bool fDeleteOnRelease);
```

Reply:
probably not necessary for P/Invoke calls

Reply:
Yes, this doesn't need the [MarshalAs]. P/Invoke calls are BOOL by default. Only COM wrappers are VARIANT_BOOL

Reply:
A stupid question, how do you tell the difference - pinvoke vs com wrapper?

Reply:
[DllImport] is a P/Invoke. [ComImport] is the COM interface definition.